physics/Interstitials/UFS_SCM_NEPTUNE/GFS_radiation_post.F90

```diff
@@ -151,96 +131,64 @@ subroutine GFS_rrtmgp_post_run (nCol, nLev, nDay, iSFC, iTOA, idxday, doLWrad, d
     if (.not. (doLWrad .or. doSWrad)) return
 
     if (doLWRad) then
-       ! #######################################################################################
-       ! Compute LW heating-rates.
-       ! #######################################################################################
+       if (do_RRTMGP) then
+          ! #######################################################################################
+          ! Compute LW heating-rates.
+          ! #######################################################################################
 
-       ! Clear-sky heating-rate (optional)
-       if (do_lw_clrsky_hr) then
-          call check_error_msg('GFS_rrtmgp_post',compute_heating_rate(  &
-               fluxlwUP_clrsky,   & ! IN  - RRTMGP upward longwave clear-sky flux profiles (W/m2)
-               fluxlwDOWN_clrsky, & ! IN  - RRTMGP downward longwave clear-sky flux profiles (W/m2)
-               p_lev,             & ! IN  - Pressure @ layer-interfaces (Pa)
-               htrlwc))             ! OUT - Longwave clear-sky heating rate (K/sec)
-       endif
+          ! Clear-sky heating-rate (optional)
+          if (do_lw_clrsky_hr) then
+             call check_error_msg('GFS_rrtmgp_post',compute_heating_rate(  &
```

Comment:
please change 'GFS_rrtmgp_post' to 'GFS_radiation_post' in order to not mislead folks if there is an error

Reply:
Done.